Cargo.toml

```diff
@@ -14,71 +14,74 @@ keywords      = ["blockchain", "wasm"]
 categories    = ["cryptography::cryptocurrencies"]
 
 [workspace.dependencies]
-alloy-dyn-abi      = { version = "0.8.9", features = ["eip712"] }
-alloy-primitives   = "0.8.9"
-anyhow             = "1"
-base64             = "0.22"
-bip32              = "0.5"
-blake2             = "0.10"
-blake3             = "1"
-bnum               = "0.12"
-borsh              = "1"
-byteorder          = "1"
-chrono             = "0.4"
-clap               = "4"
-clru               = "0.6"
-colored            = "2"
-colored_json       = "5"
-criterion          = "0.5"
-data-encoding      = "2"
-dialoguer          = "0.11"
-digest             = "0.10"
-dyn-clone          = "1"
-ed25519-dalek      = "2"
-elsa               = "1"
-glob               = "0.3"
-hex                = "0.4"
-hex-literal        = "0.4"
-home               = "0.5"
-ics23              = "0.12"
-k256               = "0.13"
-paste              = "1"
-p256               = "0.13"
-proc-macro2        = "1"
-proptest           = "1"
-prost              = "0.13"
-pyth-sdk           = "0.8.0"
-pyth-wormhole      = { git = "https://github.com/pyth-network/pyth-crosschain", package = "pyth-wormhole-attester-sdk" }
-pythnet-sdk        = "2"
-quote              = "1"
-rand               = "0.8"
-reqwest            = "0.12"
-ripemd             = "0.1"
+alloy-dyn-abi              = { version = "0.8.9", features = ["eip712"] }
+alloy-primitives           = "0.8.9"
+anyhow                     = "1"
+base64                     = "0.22"
+bip32                      = "0.5"
+blake2                     = "0.10"
+blake3                     = "1"
+bnum                       = "0.12"
+borsh                      = "1"
+byteorder                  = "1"
+chrono                     = "0.4"
+clap                       = "4"
+clru                       = "0.6"
+colored                    = "2"
+colored_json               = "5"
+criterion                  = "0.5"
+data-encoding              = "2"
+dialoguer                  = "0.11"
+digest                     = "0.10"
+dyn-clone                  = "1"
+ed25519-dalek              = "2"
+elsa                       = "1"
+glob                       = "0.3"
+hex                        = "0.4"
+hex-literal                = "0.4"
+home                       = "0.5"
+ics23                      = "0.12"
+k256                       = "0.13"
+paste                      = "1"
+p256                       = "0.13"
+proc-macro2                = "1"
+proptest                   = "1"
+prost                      = "0.13"
+pyth-sdk                   = "0.8.0"
+# The Pyth attester SDK is not published to crates.io.
+# Import from GitHub, using the latest commit at this time.
+# TODO: Ask pyth team to make a stable release and publish the crate.
+pyth-wormhole-attester-sdk = { git = "https://github.com/pyth-network/pyth-crosschain", rev = "cd67cd8" }
```

Comment:
use the crate's full name (`pyth-wormhole-attester-sdk` instead of just `pyth-wormhole`) and specify the git commit hash